### PR TITLE
Fix send_ether test (removed volatile wait_for=False condition testing)

### PR DESCRIPTION
**Pull request by badrogger**
Change ETH_TRANSFER_VALUE check logic
Refactor .travis.yml file
Remove pytest.mark.skip from manager_tests create_schain

**TIP**: Before submitting your PR, don't hesitate to remove any placeholder text, possible empty sections, and this tip.

**What**:

**Why**:

**How**:

**Checklist**:

-   [ ] Documentation
-   [ ] Tests
-   [ ] Ready to be merged

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,12 +9,18 @@ dist: bionic
...
   global:
     - NODE_OPTIONS="--max_old_space_size=4096"
+    - ETH_PRIVATE_KEY=$TEST_ETH_PRIVATE_KEY
 
 install:
   - pip install codecov pytest-cov
   - pip install -e .
   - pip install -e .[dev]
-  - export ETH_PRIVATE_KEY=$TEST_ETH_PRIVATE_KEY
+  - git clone https://${CI_USER_TOKEN}@github.com/skalenetwork/skale-manager.git
```
Comment (dmitry-tk) [inline .travis.yml]: why this moved to `install` stage?

File: tests/constants.py [hunk 2]
```diff
@@ -31,7 +32,7 @@
 ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
 
 TOKEN_TRANSFER_VALUE = 100
-ETH_TRANSFER_VALUE = 5
+ETH_TRANSFER_VALUE = Decimal('0.05')
```
Comment (dmitry-tk) [inline tests/constants.py]: we can reduce this value

File: .travis.yml (hunk 1 above)
Comment (badrogger) [reply .travis.yml]: I think contract deploying isn't part of stage test. Also travis ui compresses and hides huge npm i and truffle migrate logs

Comment (dmitry-tk) [reply .travis.yml]: doesn't it cause double deployment during the test and build stages?

**Pull request merged**